content/implementations/jvm.md

```diff
@@ -58,9 +58,9 @@ For example:
 
 ```xml
 <dependency>
-    <groupId>info.cukes</groupId>
+    <groupId>io.cucumber</groupId>
     <artifactId>cucumber-java</artifactId>
-    <version>{{ site.versions.cucumber_jvm }}-SNAPSHOT</version>
+    <version>2.1.1-SNAPSHOT</version>
```

Comment:
Shouldn't this use  site.versions.cucumber_jvm ?

Reply:
Should that show the current version? That doesn't seem to work atm (nor on the current site).

Reply:
You might have a look at this: https://gohugo.io/variables/ could be in there I think